README.md

```diff
@@ -13,134 +13,234 @@
 
 In our curriculum, we give preference to MOOC (Massive Open Online Course) style courses because these courses were created with our style of learning in mind.
 
-## Curricular Guideline
 
-OSSU Data Science uses the report [Curriculum Guidelines for Undergraduate Programs in Data Science](https://www.amstat.org/asa/files/pdfs/EDU-DataScienceGuidelines.pdf) as our guide for course recommendation.
+# Curricular Guideline
 
-## Curriculum
+OSSU Data Science uses the report [Curriculum Guidelines for Undergraduate Programs in Data Science](https://www.amstat.org/asa/files/pdfs/EDU-DataScienceGuidelines.pdf) as our guide for course recommendation.
 
----
 
-### Introduction to Data Science
-[What is Data Science](https://www.coursera.org/learn/what-is-datascience)
+# Curriculum
 
-### Introduction to Computer Science
-_Students who already know basic programming in any language can skip this first course_
 
-[Python for Everybody](https://www.py4e.com/) 
+## Prerequisites
 
-[Introduction to Computer Science and Programming Using Python](https://www.edx.org/course/introduction-computer-science-mitx-6-00-1x-7)
+The Data Science curriculum assumes the student has taken [high school math](https://github.com/ossu/computer-science/blob/master/FAQ.md#how-can-i-review-the-math-prerequisites) and [statistics](https://www.khanacademy.org/math/probability).
 
-[Introduction to Computational Thinking and Data Science](https://www.edx.org/course/introduction-computational-thinking-data-mitx-6-00-2x-3)
 
-### Data Structures and Algorithms
-_The Algorithms courses are taught in Java. If students need to learn Java, they should take this course first_
+## Intro to Data Science
 
-[Java Programming](https://java-programming.mooc.fi/)
+**Topics covered**:
+`simple programs`
+`simple data structures`
 
-[Algorithms, Part I](https://www.coursera.org/learn/algorithms-part1)
+Courses | Duration | Effort | Prerequisites | Discussion
+:-- | :--: | :--: | :--: | :--:
+[What is Data Science](https://www.coursera.org/learn/what-is-datascience) | 1 weeks | 8 hours/week | none | [chat]()
 
-[Algorithms, Part II](https://www.coursera.org/learn/algorithms-part2)
 
-### Databases
-[Database Management Essentials](https://www.coursera.org/learn/database-management)
+## Intro CS
 
-[Data Warehouse Concepts, Design, and Data Integration](https://www.coursera.org/learn/dwdesign)
+### Introduction to Programming
 
-[Relational Database Support for Data Warehouses](https://www.coursera.org/learn/dwrelational)
+If you've never written a for-loop, or don't know what a string is in programming, start here. This course is self-paced, allowing you to adjust the number of hours you spend per week to meet your needs.
 
-[Business Intelligence Concepts, Tools, and Applications](https://www.coursera.org/learn/business-intelligence-tools)
+**Topics covered**:
```

Comment:
Maintaining these topic lists properly is some amount of work and I have no evidence that students benefit from the addition.